readline.h

```diff
@@ -1,5 +1,5 @@
 /*
-    Replace libreadline.
+    Replace libreadline
```

Comment:
This is all or nothing. Either the headers of all files are changed or none. I just made them consistent. Please use these lines with a dot at the end for now and also place it for the new files. This way we can do the "with-or-without-dot-discussion" later on.